app.js

```diff
@@ -53,9 +53,18 @@ window.AppData = new Proxy(defaultData, {
 //   });
 // }, 8000);
 
+function padNum(num) {
+  // input '1' => output '01'
+  return ('00' + num).substr(-2,2);
+}
+
 function formatTime(c, d) {
   // example:   0:35 / 2:33
-  return ((c / 60).toFixed(2) + ' / ' + (d / 60).toFixed(2)).replace('.', ':');
+  const cMin = Math.floor(c / 60);
+  const cSec = Math.floor(c - cMin * 60);
+  const dMin = Math.floor(d / 60);
+  const dSec = Math.floor(d - dMin * 60);
+  return `${cMin}:${padNum(cSec)} / ${dMin}:${padNum(dSec)}`;
```

Comment:
Not sure if you want to maybe break this down one more level since it looks pretty duplicated.

```js
function formatTime(current, total) {
  const padNum = (num) => {
    return ("00" + num).substr(-2);
  };
  // example:   0:35 / 2:33
  const duration = (time) => {
    const minutes = Math.floor(time / 60);
    const seconds = Math.floor(time - (minutes * 60));
    return `${ minutes }:${ padNum(seconds) }`;
  };
  return `${ duration(current) } / ${ duration(total) }`;
}

console.log(formatTime(6543, 7387)); // Output: 109:03 / 123:07
```

My variable names are kind of :poop:, but hey. Plus we also may want to test it with longer videos, like https://www.youtube.com/watch?v=HflDxjLHDNU (which is 2:03:07). Not sure if that would display as 123:07, or if we want to add more logic for potential hour long videos so it displays as 2:03:07.